documentation/airport-file-standards.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Aircraft File Standards
+---
+[back to index](index.html)
+
+## Overview
+This document serves as a checklist for airport contributors to review to ensure their airport file is "up to snuff". There are two levels of airports: `Standard` and `Premium`, the latter designator being used only for our most pristine, complete, realistic, and well-documented airports. In order for a new airport to be merged, it must at least meet all the `Standard` specifications listed in the section below. If it does not meet these requirements, it will require extra work before being merged. It will often require a team of people to complete work on an airport, because of the wide variety of tasks to complete.
+
+If you ever need help working on an airport, the best place to go is the [openScope Slack](https://openscopeatc.slack.com) ([click here if you need an invite](http://slack.openscope.co)). There is always somebody available, and many who will be glad to walk you through anything you may need help with.
+
+If you have not contributed before, please note that we will need to add you to the organization so you can keep your branch _directly_ on the openscope repository. We do all branching and pull requests internally like this, instead of using forks. Join slack and request to be invited to the openScope team by `@erikquinn`.
+
+---
+
+## Standard Requirements
+### Terrain GeoJSON File
+Even if it is empty, a terrain file must be included as part of the pull request. We ask that the qGIS files used in the generation of the terrain GeoJSON file also be uploaded to the [openscope/other-files](https://github.com/openscope/other-files) repository. This way, if an airspace boundary is changed, we can regenerate the terrain file in far less time. This takes a long time to generate, and is quite a pain, but there are people on slack who would be happy to do it for you.
+
+### Polygonal Airspace Boundary
+An airspace boundary must be defined with GPS coordinates. Previously, we allowed just a "airspace center" and "airspace radius", but now there must be a defined edge to the airspace.
+
+### All SIDs & STARs
+All procedures for a given airport must be included in the pull request. Not all have to be _used_ for anything, but (to the extent possible) all SIDs and STARs must be present in the file.
+
+### Accurate traffic level (and info to back it up)
+The traffic level must be commensurate with real-world levels, with the arrival and departure rates. Airports can be very quiet during some hours, and very busy during others-- therefore, the traffic included in the airport file shall be:
+- not less than `yearly operations / 8760` (hourly equivalent of yearly rate)
```

Comment:
math is hard.  are you saying here that the minimum should be no less than the average aircraft per hour averaged over a year?